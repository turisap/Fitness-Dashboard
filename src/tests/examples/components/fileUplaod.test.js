/**
 * Created by HP on 08-Jan-18.
 */
import React from 'react';
import { shallow } from 'enzyme';
import FileUpload from '../../components/FileUpload';


test('should render the file upload component properly', () => {
    const wrapper = shallow(<FileUpload/>);
    expect(wrapper).toMatchSnapshot();
});