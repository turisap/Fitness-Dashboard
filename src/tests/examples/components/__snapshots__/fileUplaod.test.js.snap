// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the file upload component properly 1`] = `
<Upload
  accept=""
  action="action for file upload"
  beforeUpload={[Function]}
  className="avatar-uploader"
  data={Object {}}
  disabled={false}
  listType="picture-card"
  multiple={false}
  name="avatar"
  onChange={[Function]}
  prefixCls="ant-upload"
  showUploadList={false}
  supportServerRender={true}
  type="select"
>
  <div>
    <Icon
      type="plus"
    />
    <div
      className="ant-upload-text"
    >
      Upload
    </div>
  </div>
</Upload>
`;
